docs/install/build.mdx

```diff
@@ -33,24 +45,61 @@ Depending on your platform, you may also need to install additional
 dependencies to build Ghostty. For Linux, see [Linux](#linux). For macOS,
 see [macOS](#macos).
 
-Begin by cloning the Ghostty repository and changing into the directory:
+## Getting the source code
 
-```sh
-git clone https://github.com/ghostty-org/ghostty
-cd ghostty
+Begin by **downloading Ghostty's source tarball**.
+
+<Warning>
+
+While you can compile Ghostty from source by checking out its Git repository,
+this is **not recommended** for anyone but Ghostty developers. This is because
+source tarballs contain various preprocessed files that make Ghostty require far
+fewer dependencies to compile.
+
+If you choose to compile with a Git source, you need to install extra
+dependencies, which will be covered in the build instructions.
```

Comment:
These are specifically covered (or meant to be covered) in the README, since that is associated with the Git source. We can update a link to that here. The website covers source tarballs, the README covers Git source.